<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="73054" started-at="2020-01-06T07:59:27Z" finished-at="2020-01-06T08:00:40Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="73054" started-at="2020-01-06T07:59:27Z" finished-at="2020-01-06T08:00:40Z">
      <class name="com.qa.Eprocbilling.testCases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="14268" started-at="2020-01-06T13:29:27Z" finished-at="2020-01-06T13:29:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="LoginPagetitleTest()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="LoginPagetitleTest" duration-ms="28" started-at="2020-01-06T13:29:41Z" finished-at="2020-01-06T13:29:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [javaclassesinpune.com] but found [webERP Login screen]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [javaclassesinpune.com] but found [webERP Login screen]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.qa.Eprocbilling.testCases.LoginPageTest.LoginPagetitleTest(LoginPageTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPagetitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="745" started-at="2020-01-06T13:29:41Z" finished-at="2020-01-06T13:29:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="12805" started-at="2020-01-06T13:29:42Z" finished-at="2020-01-06T13:29:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="invalidLoginPageTest()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="invalidLoginPageTest" duration-ms="431" started-at="2020-01-06T13:29:54Z" finished-at="2020-01-06T13:29:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginPageTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="1293" started-at="2020-01-06T13:29:55Z" finished-at="2020-01-06T13:29:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="14213" started-at="2020-01-06T13:29:56Z" finished-at="2020-01-06T13:30:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidpasswordTest()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="invalidpasswordTest" duration-ms="765" started-at="2020-01-06T13:30:10Z" finished-at="2020-01-06T13:30:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [incorrect password
The user/password combination
is not a valid user of the system] but found [incorrect password
The user/password combination
is not a valid user of the system]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [incorrect password
The user/password combination
is not a valid user of the system] but found [incorrect password
The user/password combination
is not a valid user of the system]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.qa.Eprocbilling.testCases.LoginPageTest.invalidpasswordTest(LoginPageTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidpasswordTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="844" started-at="2020-01-06T13:30:11Z" finished-at="2020-01-06T13:30:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="12940" started-at="2020-01-06T13:30:12Z" finished-at="2020-01-06T13:30:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="logoTest()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="logoTest" duration-ms="89" started-at="2020-01-06T13:30:25Z" finished-at="2020-01-06T13:30:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="828" started-at="2020-01-06T13:30:25Z" finished-at="2020-01-06T13:30:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="12791" started-at="2020-01-06T13:30:26Z" finished-at="2020-01-06T13:30:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validloginPageTest()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="validloginPageTest" duration-ms="9" started-at="2020-01-06T13:30:39Z" finished-at="2020-01-06T13:30:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validloginPageTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.Eprocbilling.testCases.LoginPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="957" started-at="2020-01-06T13:30:39Z" finished-at="2020-01-06T13:30:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.Eprocbilling.testCases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
